.github/workflows/scheduled_sweep.yml

```diff
@@ -0,0 +1,14 @@
+---
+on:
+  schedule:
+    - cron: '14 7 15 * *'
+name: Scheduled sweep
+jobs:
+  clippy:
+    name: Sweep
+    runs-on: [self-hosted, heavy]
+    steps:
+      - name: Clone the repository
+        uses: actions/checkout@v2
+      - name: Run sweep
+        run: cargo make sweep
```

Comment:
As we discussed before, there's no need to do sweep, because target folder is cleaned up each time anyway